packages/snaps-controllers/src/interface/SnapInterfaceController.test.tsx

```diff
@@ -29,14 +37,89 @@ jest.mock('@metamask/snaps-utils', () => ({
 }));
 
 describe('SnapInterfaceController', () => {
+  it('handles a notificationsListUpdated event on the controller messenger', async () => {
+    const rootMessenger = getRootSnapInterfaceControllerMessenger();
+    const controllerMessenger =
+      getRestrictedSnapInterfaceControllerMessenger(rootMessenger);
+
+    const controller = new SnapInterfaceController({
+      messenger: controllerMessenger,
+      state: {
+        interfaces: {
+          // @ts-expect-error missing properties
+          '1': {
+            contentType: ContentType.Notification,
+          },
+          // @ts-expect-error missing properties
+          '2': {
+            contentType: ContentType.Dialog,
+          },
+        },
+      },
+    });
+
+    rootMessenger.publish(
+      'NotificationServicesController:notificationsListUpdated',
+      [],
+    );
+
+    // defer
+    setTimeout(() => {
```

Comment:
It's unclear to me why this is necessary 🤔

Reply:
Not needed, you're right.